### DISCUSSION TITLE
Lesson 13 Full stack DeFi : Header.tsx

### DISCUSSION THREAD

**Discussion by NoroMeb** [smartcontractkit/full-blockchain-solidity-course-py#1657] [Q&A]
Hello, 
im new in JS/TS and in Header.tsx when calling useStyles() `const classes = useStyles()` declared up `const useStyles = makeStyles((theme: any) => ({
    container: {
        padding: theme.spacing(4),
        display: "flex",
        justifyContent: "flex-end",
        gap: theme.spacing(1)
    },
}));`

i have this errore message `This expression is not callable.
  Type 'never' has no call signatures.`

**Comment by cromewar** [comment]
Hello @NoroMeb 
The `useStyles` method is deprecated and you should not use it anymore, you can follow this for reference: https://stackoverflow.com/questions/56554586/how-to-use-usestyle-to-style-class-component-in-material-ui

Also, remember the lesson 13 is a bonus and actually some of the libraries used on that lesson are also deprecated so I would actually suggest not take that lesson and learn some front with the JS edition which is also in the FreeCodeCamp Channel.
